addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -160,9 +156,12 @@ def format_alphanumeric(text_to_convert):
         # Constraints within the edi make local rounding on price included taxes a problem.
         # To solve this there is a <Arrotondamento> or 'rounding' field, such that:
         #   taxable base = sum(taxable base for each unit) + Arrotondamento
-        tax_details = self._prepare_edi_tax_details(
-            filter_to_apply=lambda l: l['tax_repartition_line_id'].factor_percent >= 0
-        )
+        def filter_details(l):
+            return (l['tax_repartition_line_id'].factor_percent >= 0
+                    and l['tax_id'].amount >= 0
+                    and not l['tax_id'].l10n_it_pension_fund_type)
```

Comment:
same as above...
are you sure we can do this? `l10n_it_pension_fund_type` is from the new module

Reply:
Yeah, this is just wrong. Strange, I must have messed up the commit.